### PR TITLE
Remove swipe refresh

### PR DESCRIPTION
**Pull request by konifar**
* Removed SwipeRefreshView.
* Added sessions json data.
* Fixed session item layout.

### CODE REVIEW

**Comment by hotchemi** [comment]
LGTM

**Comment by hkurokawa** [comment]
Just wondering why `fetchAndSave()` is necessary.  Maybe to refresh the local data in case the remote data is updated?

**Comment by konifar** [comment]
@hkurokawa Yes. As you said, this is because to refresh the local data.

I think it is better that the user should not think reload.

File: app/src/main/res/raw/sessions_ja.json [hunk 1]
```diff
@@ -0,0 +1,1082 @@
...
+    "speaker": {
+      "id": 1,
+      "name": "wasabeef",
+      "image_url": "https://pbs.twimg.com/profile_images/427481863343452160/i-G-x-Gw.jpeg",
+      "twitter_name": "wasabeef_jp",
+      "github_name": "wasabeef_jp"
+    },
+    "stime": "2015-02-18 10:00:00",
```
Comment (hotchemi) [inline app/src/main/res/raw/sessions_ja.json]: 2016-02-18 right?

Comment (hotchemi) [inline app/src/main/res/raw/sessions_ja.json]: 2016-02-18 right?

Comment (konifar) [inline app/src/main/res/raw/sessions_ja.json]: Oh, my mistake! I'll fix after go back home.

File: app/src/main/res/raw/sessions_ja.json [hunk 2]
```diff
@@ -0,0 +1,1082 @@
...
+    "category": {
+      "id": 1,
+      "name": "基調講演"
+    },
+    "place": {
+      "id": 1,
+      "name": "基調講演会場"
+    },
+    "language_id": "ja",
+    "slide_url": ""
```
Comment (hotchemi) [inline app/src/main/res/raw/sessions_ja.json]: Is this prop needed?

**Comment by hotchemi** [comment]
@konifar Reviewed session json, great works!

File: app/src/main/res/raw/sessions_ja.json (hunk 2 above)
Comment (konifar) [inline app/src/main/res/raw/sessions_ja.json]: It is not used now. I fill this props value after sessions finished. The user can check slide in session detail page in app.

Comment (hotchemi) [inline app/src/main/res/raw/sessions_ja.json]: LGTM!

**Pull request merged**